scripts/beegfspkgs.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+KERNEL=$(uname -r)
+echo "Kernel version is $KERNEL"
+#
+systemctl stop firewalld
+systemctl disable firewalld
+#
+sudo yum -y install epel-release
```

Comment:
Isn't the script run with sudo?  If so, you don't need sudo throughout the script

Reply:
yes, made that change.